src/network/networkd-route.c

```diff
@@ -2573,17 +2573,17 @@ int config_parse_route_hop_limit(
         r = safe_atou32(rvalue, &k);
         if (r < 0) {
                 log_syntax(unit, LOG_WARNING, filename, line, r,
-                           "Could not parse route hop limit %s \"%s\", ignoring assignment: %m", lvalue, rvalue);
+                           "Could not parse per route hop limit, ignoring assignment: %s", rvalue);
```

Comment:
I think that we have a general anti-pattern here: we should just say "Could not parse %s.%s …", section, lvalue". This is both easier for us (because we don't have to craft a custom error message), *and* easier for the user, because they don't have to try to guess what setting we were trying to describe.

I think we should rework most error messages like this. But we should probably do that in one big pull request. So let's merge this, since it fixes other issues too.

Reply:
I agree. I will do that later.